### PR TITLE
Fix inconsistent wording in settings page

### PR DESCRIPTION
**Pull request by Derkades**
a few things I noticed while translating

### CODE REVIEW

**Comment by matiasdelellis** [comment]
Hi @Derkades 
Thank you very much for these corrections.
In general, I think you are right, but if you are not offended, I would rather postpone this PR for later. I want to release a minor revision these days, and if I apply this patch, there will be many incomplete translations. Then we can give the rest of the translators more time to update.

File: templates/settings/admin.php [hunk 1]
```diff
@@ -47,19 +47,19 @@
...
 			<span><input type="range" id="min-confidence-range" min="0.0" max="1.1" value="0.99" step="0.01" class="ui-slider"></span>
-			<label for="min-confidence-range"><?php p($l->t('Highest minimum confidence'));?></label>
+			<label for="min-confidence-range"><?php p($l->t('Higher minimum confidence'));?></label>
 			<span id="min-confidence-value"class="span-highlighted">...</span>
 			<a id="restore-min-confidence" class="icon-align icon-history" style="display: none;" title="<?php p($l->t('Restore'));?>" href="#"></a>
 			<a id="save-min-confidence" class="icon-align icon-edit" style="display: none;" title="<?php p($l->t('Save'));?>" href="#"></a>
 		</p>
 		<br>
 		<h3>
-			<?php p($l->t('Configuration information'));?>
```
Comment (matiasdelellis) [inline templates/settings/admin.php:59]: Here should be more information, such as the version of pdlib, if CUDA it is enabled, etc. So, don't change it. :wink:

**Comment by Derkades** [comment]
Of course, merge this PR at any time you want or not at all.

**Comment by matiasdelellis** [comment]
Thanks again @Derkades

**Comment by Derkades** [comment]
These translations haven't updated for me on transifex, am I missing something?

**Comment by matiasdelellis** [comment]
Hi @Derkades 
Tomorrow add that.. :wink:

**Pull request merged**